_hacks/34.md

```diff
@@ -1,5 +1,15 @@
 ---
 hack_number: 34
 date: 2023-02-25
-upcoming: true
+upcoming: false
 ---
+
+It was chaos without our leaders: we met up at 9:30 and said what we'd do, people then hacked and chatted, and we demoed at 16:30. No podcasts were recorded.
+
+## Things people did
+
+- **Piers** blew up a microcontroller for his oil sensor and worked out that he had ground and power flipped (a big no-no)
+- **Dan** built a standing desk, fixed his mics sound, and used a new data format to show the current and next shows on TV on the web out of thin air (or from the digital TV guide signal)
+- **Joe** was adding HTLM and CSS to his TiddlyWiki static site export to make it look nice
+- **Hugh** read up a lot about ansible
+- **Panda** worked on some websites and solved a complex problem by simply buying a new domain name
```

Comment:
- **Adam**  completely failed ro open an LLX, butbrje. Scuppered that because he misread th eifns sg Gatwick snd had ro book an emergency 6 hour train tit he Norfolk.

Reply:
Oh gosh...birthday drinks tried to comment on this PR, I see! Well that can safely be ignored, methinks!